draft-ietf-rtcweb-jsep.xml

```diff
@@ -93,17 +93,21 @@ ipr="trust200902">
         <t>With these considerations in mind, this document describes
         the Javascript Session Establishment Protocol (JSEP) that
         allows for full control of the signaling state machine from
-        Javascript. JSEP removes the browser almost entirely from the
-        core signaling flow, which is instead handled by the Javascript
-        making use of two interfaces: (1) passing in local and remote
+        Javascript. As described above, JSEP assumes a model in which
+        a Javascript application runs inside a runtime containing
+        WebRTC APIs (the "JSEP implementation"). The JSEP implementation
+        almost entirely divorced from the core signaling flow, which is instead
+        handled by the Javascript making use of two interfaces: (1) passing in local and remote
         session descriptions and (2) interacting with the ICE state
-        machine.</t>
+        machine. The combination of the JSEP implementation and the Javascript
+        application is referred to throughout this document as a "JSEP endpoint." </t>
 
         <t>In this document, the use of JSEP is described as if it
-        always occurs between two browsers. Note though in many cases
-        it will actually be between a browser and some kind of server,
-        such as a gateway or MCU. This distinction is invisible to the
-        browser; it just follows the instructions it is given via the
+        always occurs between two JSEP endpoints. Note though in many cases
+        it will actually be between a JSEP endpoint and some kind of
+        server, such as a gateway or MCU. As long as the server
+        generates JSEP-compliant SDP, this distinction is invisible to the
```

Comment:
Is this JSEP-compliant SDP comment accurate? We will handle SDP even that doesn't conform to what a JSEP implementation would generate, as long as it meets our mandatory-to-use statutes.